Characterbranch

paxelpixel opened the pull request:
add initial gui, eveyrthing is subject to change

paxelpixel committed:
test

paxelpixel committed:
added CharacterView and ViewModel Classes

paxelpixel committed:
Initial design

xenonsin commented:
Can you create a screen shot, since I can't really review the code cause its all GUI. Please :D

paxelpixel committed:
removed Initial Gui

added a page per tab functionality which means there is a seperate page
file for a different tab to avoid huge amount of gui code in one main
file. so we can work seperately for different tabs

paxelpixel committed:
very small UI changes
